packages/system-tests/src/spectron/application.ts

```diff
@@ -111,9 +111,23 @@ export class SpectronApplication {
     );
   }
 
+  private getKeybindingPlatform(): string {
+    switch (process.platform) {
+      case 'darwin':
+        return 'osx';
+      case 'win32':
+        return 'win';
+      default:
+        return process.platform;
+    }
+  }
+
   private retrieveKeybindings() {
     fs.readFile(
-      path.join(process.cwd(), `test_data/keybindings.json`),
+      path.join(
+        process.cwd(),
+        `test_data/keybindings.${this.getKeybindingPlatform()}.json`
```

Comment:
Read directly from the bindings stored in test_data. `getKeybindingPlatform` will retrieve the bindings for the correct OS.

Reply:
@Sweetman Actually, I'm surprised that this works. You hardcoded the '/' in the second part.